packages/frontend/src/lib/table/model/ModelColumnName.svelte

```diff
@@ -8,9 +8,18 @@ function openDetails() {
 }
 </script>
 
-<button class="flex flex-col w-full" title="{object.name}" on:click="{() => openDetails()}">
-  <div class="text-sm text-gray-300 overflow-hidden text-ellipsis w-full">
+<button
+  class="flex flex-col w-full"
+  title="{object.name}"
+  on:click="{() => openDetails()}"
+  aria-label="Open Model Details">
+  <div class="text-sm text-gray-300 overflow-hidden text-ellipsis w-full" aria-label="Model Name">
```

Comment:
```suggestion
  <div class="text-sm text-gray-300 overflow-hidden text-ellipsis w-full text-left" aria-label="Model Name">
```

When the name is too short we have some weird alignements

![image](https://github.com/containers/podman-desktop-extension-ai-lab/assets/42176370/9f7f8698-121d-4ee1-bca4-583cb1f1d04b)